zilliqa/src/contracts/deposit_v1.sol

```diff
@@ -137,7 +137,7 @@ contract DepositInit is UUPSUpgradeable {
         uint256 _maximumStakers,
         uint64 _blocksPerEpoch,
         InitialStaker[] memory initialStakers
-    ) public initializer {
+    ) public payable initializer {
```

Comment:
Don't we need to change v2 as well?

Reply:
v2 doesn't seem to have this function.

Reply:
v1 does the initialising, further versions inherit the previous version's storage and so do not need to initialise.

Reply:
Makes sense, thanks for explaining.